client/app/components/basic_components.coffee

```diff
@@ -125,7 +125,41 @@ FormButtons = React.createClass
 
         div null,
             div className: 'col-sm-offset-4',
-                FormButton formButton for formButton in @props.buttons
+                for formButton, index in @props.buttons
+                    formButton.key = index
+                    FormButton formButton
+
+MenuItem = React.createClass
```

Comment:
I made a few changes in the toolbar & toolbox components so that they always delegate the actual action to their parent component, because it depends on which view is showing them. However, i could not make them below 80 char, so I made the MenuItem & cie components to clean up these components. (https://github.com/cozy/cozy-emails/pull/391/files#diff-d3d0c90ae3619da533428714ee04cd31R44)